src/tools/Makefile.am

```diff
@@ -91,53 +91,41 @@ gids_tool_SOURCES = gids-tool.c util.c
 gids_tool_LDADD = $(OPTIONAL_OPENSSL_LIBS)
 
 npa_tool_SOURCES = npa-tool.c fread_to_eof.c util.c npa-tool-cmdline.c
-npa_tool_LDADD = $(top_builddir)/src/libopensc/libopensc.la \
-				 $(OPENPACE_LIBS)
-npa_tool_CFLAGS = -I$(top_srcdir)/src $(OPENPACE_CFLAGS) $(OPENSSL_CFLAGS)
-npa_tool_CFLAGS += -Wno-unused-but-set-variable
```

Comment:
I think disabling this warning is needed, because otherwise gengetopt generates code with warnings. and it should be the same for the other tools that are parsing the command line with gengetopt

Reply:
If this is the case, it needs at least the comment why this exists. But given that the CI does not prompt any errors now, it might be that they are no longer needed. Do you have at hand on what configuration (compiler, os, ...) this was failing?

Reply:
Merging. If you will figure out these flags are missing for some reason, please add them back with proper reasoning in a new PR.